projects/fsastorefrontlib/src/cms-components/comparison-table/comaprison-table-panel-item/comparison-table-panel-item.component.html

```diff
@@ -1,6 +1,9 @@
 <ng-container *ngIf="product$ | async as product; else loading">
   <ng-container *ngIf="!isLoading; else loading">
-    <div class="table-header">
+    <div
+      class="table-header"
+      [ngClass]="recommendedProduct == product.code ? 'highlight' : ''"
```

Comment:
Instead of adding `highlight` class, maybe we can use existing `border border-color-6`. Since it will probably be changed anyway, there will be one less thing to remove from scss later on.

Reply:
Class removed